pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -77,6 +77,10 @@ func doUpdate(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool) {
 			return changed, false
 		}
 	}
+	if status.State == types.VERIFIED && status.ObjType == types.BaseOsObj {
+		// For base images, we don't need to create volumes
+		return changed, true
+	}
```

Comment:
We shouldn't put in a hack like this.
Instead we can wait until we have ContentTree support and have baseosmgr ask for a CT instead of a Volume.

Reply:
Makes sense.